src/librbd/cache/rwl/LogEntry.h

```diff
@@ -106,6 +108,9 @@ class GenericWriteLogEntry : public GenericLogEntry {
   std::shared_ptr<SyncPointLogEntry> get_sync_point_entry() override {
     return sync_point_entry;
   }
+  virtual void copy_pmem_bl(bufferlist *out_bl) {
```

Comment:
Nit: why does this need to be exposed at the base class instead of the `GenericWriteLogEntry` level?

Reply:
@dillaman This had already been exposed at  `GenericWriteLogEntry`.

Reply:
Sorry -- it was bad folding in the GitHub review. This should be pure virtual, then.

Reply:
@dillaman made it as pure virtual.